website/static/js/pages/profile-settings-applications-list.js

```diff
@@ -0,0 +1,7 @@
+'use strict';
+
+var viewModels = require('../apiApplication');
+
+var ctx = window.contextVars;
+var apiApplication = new viewModels.ApplicationsList('#appList', ctx.urls);
+ctx.apiApplication = apiApplication;
```

Comment:
Originally used it for debugging purposes; I'll remove this line for production.